index.html

```diff
@@ -102,51 +102,97 @@ <h1 id="title">Dubbing and Audio description Profiles of TTML2</h1>
-        Alternatively, styling can be applied to assist voice artists when recording scripted dialogue or <a>audio descriptions</a>.
-      </p>
-      <p>Creating audio description content is also a complex process with similar steps.
-        An <dfn>audio description</dfn>, also known as video description, is an audio service
-        to assist viewers who can not fully see a visual presentation to understand the content.
-        It is the result of the audio rendition of one or more <a>descriptions</a>
-        mixed with the audio associated with the programme prior to any mixing with audio description
-        (sometimes referred to as <dfn>main programme audio</dfn>),
-        at moments when this does not clash with dialogue, to deliver an <dfn>audio description mixed audio track</dfn>.
-        A <dfn>description</dfn> is a set of words that describe an aspect of the programme presentation,
-        suitable for rendering into audio by means of vocalisation and recording
-        or used as a text alternative source for text to speech translation, as defined in [[WCAG20]].
-        More information about what <a>audio description</a> is and how it works can be found at [[WHP051]].
-      </p>
-
-      <p class="issue" data-number="28"></p>
-      <p>This specification defines DAPT, a TTML-based format for the exchange of timed text content among authoring and prompting tools in the localization and <a>audio description</a> pipeline.
-        A DAPT file is designed to carry pertinent information for dubbing or <a>audio description</a> such as type of script, dialogue, descriptions, timing, metadata, original language text, transcribed text, language information, and to be extensible for future annotations.
-        This specification first defines the data model (see [[[#data-model]]]) for DAPT scripts and then its representation as a TTML document with restrictions (see [[[#ttml-format]]]).</p>
+      <section id="intro-overview">
+        <h3>Transcripts</h3>
+        <p>DAPT is a TTML-based transcript format for the exchange of timed text content
+          among authoring, prompting and playback tools in the localization and <a>audio description</a> pipelines.
+          A DAPT file is designed to carry pertinent information for dubbing or <a>audio description</a>
+          such as type of script, dialogue, descriptions, timing, metadata, original language transcribed text, translated text, language information, and audio mixing instructions,
+          and to be extensible to allow user-defined annotations or additional future features.</p>
+        <p>This specification defines the data model for <a>DAPT scripts</a> and
+          its representation as a [[TTML2]] document (see [[[#data-model]]])
+          with some constraints and restrictions (see [[[#profile-constraints]]]).</p>
+        <p>A <a>DAPT script</a> is expected to be used to make audio visual media accessible
```

Comment:
"make ... accessible" could be understood as "DAPT is used in subtitle or closed captioning pipelines". This is not directly the case. Should we highlight that?

Reply:
I don't understand - does the following text not do that already, including the words immediately following: "or localized..."? Could you explain more about the concern?